api/v1alpha1/release_webhook.go

```diff
@@ -17,38 +17,116 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"fmt"
-	"k8s.io/apimachinery/pkg/runtime"
+	"context"
+	"encoding/json"
 	"reflect"
+	"strings"
+
+	"gomodules.xyz/jsonpatch/v2"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-func (r *Release) SetupWebhookWithManager(mgr ctrl.Manager) error {
-	return ctrl.NewWebhookManagedBy(mgr).
-		For(r).
-		Complete()
-}
+const authorLabel = "release.rhtap.openshift.io/author"
```

Comment:
I guess that Leandro's PR was not merged by the time you added this. Could you move it to the right place?

Reply:
I merged first so he moved it as part of his pr (which is now merged)